app/src/androidTest/java/org/simple/clinic/di/TestRetrofitModule.kt

```diff
@@ -37,7 +37,7 @@ class TestRetrofitModule {
       moshi: Moshi,
       okHttpClient: OkHttpClient
   ): Retrofit {
-    val baseUrl = BuildConfig.FALLBACK_ENDPOINT
+    val baseUrl = BuildConfig.MANIFEST_ENDPOINT
```

Comment:
They both have the same link to it? Can you tell me what is the difference?

Reply:
The explanation is in `gradle.properties` if you look for the `qaFallbackApiEndpoint` property.